src/components/navbar.tsx

```diff
@@ -31,42 +32,65 @@ const Navbar = () => {
     { name: dict.navbar.projects, path: "/projects" },
   ];
 
+  const isMemberOrCollaborator =
+    pathname === "/about/become-a-member" ||
+    pathname === "/about/become-a-collaborator";
+
+  const navbarBackgroundColor =
+    pathname === "/about/become-a-member"
+      ? "bg-primary"
+      : pathname === "/about/become-a-collaborator"
+        ? "bg-blue"
+        : "bg-background";
+  const linkColor = isMemberOrCollaborator ? "text-frost" : "text-gray";
+  const currentLink = isMemberOrCollaborator ? "text-white" : "text-black";
+  const colorLogo = isMemberOrCollaborator ? "white" : "#ED7950";
+  const hamburgerMenuColor = isOpen
+    ? "text-gray"
+    : isMemberOrCollaborator
+      ? "text-white"
```

Comment:
```suggestion
      ? "text-white/50"
```

It has a bit of opacity 👀 
![image](https://github.com/user-attachments/assets/079b1699-78af-401b-b681-16e67e2ba75c)
![image](https://github.com/user-attachments/assets/f1fd59bf-9694-4d8b-93e2-bf7b1ebdd41b)